tomviz/MainWindow.cxx

```diff
@@ -382,6 +382,21 @@ QString MainWindow::readScript(const QString &scriptName)
     QByteArray array = file.readAll();
     return QString(array);
   }
+// On OSX the above doesn't work in a build tree.  It is fine
+// for superbuilds, but the following is needed in the build tree
+// since the executable is three levels down in bin/tomviz.app/Contents/MacOS/
+#if __APPLE__
+  else
+  {
+  path = QApplication::applicationDirPath() + "/../../../../share/tomviz/scripts/";
```

Comment:
...add a line here `qDebug() << QApplication::applicationDirPath();` and tell me what it says (you can shorten it to start with .../tomviz_build if you want).